doc/sphinx/source/backends/backend-api/index.rst

```diff
@@ -4,7 +4,29 @@
 Backend
 =======
 
+* :ref:`generic_backend`
+* :ref:`intelgpu_backend`
+
+
+.. _generic_backend:
+
+Generic backend
```

Comment:
Not sure if `generic backend` is the best term for it. Maybe `backend interface`.